packages/documentation/.storybook/manager-head.html

```diff
@@ -2,6 +2,35 @@
 <script src="/assets/scripts/analytics-helper.js"></script>
 <script src="/assets/scripts/analytics-events.js"></script>
 
+<!-- Open Graph general (Facebook, Pinterest & Google+) -->
+<meta property="og:type" content="website" />
+<meta property="og:title" content="Swiss Post Design System" />
+<meta property="og:locale" content="en_US" />
+<meta
+  property="og:description"
+  content="The Swiss Post Design System pattern library for a unified and accessible user experience across the web platform."
+/>
+<meta property="og:url" content="https://next.design-system.post.ch/" />
+<meta
+  property="og:image"
+  content="https://github.com/swisspost/design-system/packages/documentation/public/images/DesignSystemPreview1280x640.png"
```

Comment:
It would be better use https://next.design-system.post.ch/images as image source instead of GitHub.

Reply:
@b1aserlu I don't think that the resource is that critical, so it's not a blocker that it will work only when released. What could be more an issue is that we forget to change the URL when we'll change next.design-system.post.ch to design-system.post.ch. Maybe you could have a look, how to inject our base URL. It could be, for example, with environment variables. Can you think of alternative ways?

Reply:
I try it with JS now. But I need the preview to work so I can check if it works.

Reply:
Solving it with JS might be a problem for bots that will try to parse the HTML for this information. I'm having doubt that it will work.

Reply:
You are right, it did not work. But I am out of useful ideas how to do it.